app/person.py

```diff
@@ -0,0 +1,24 @@
+class Person:
+	"""An overal class for person"""
+
+	def __init__(self, person_name):
+
+		self.person_name = person_name
+
+	def __repr__(self):
+ 		return self.person_name
+
+
+class Fellow(Person):
+	"""A class for fellows"""
+
+	def __init__(self, person_name):
+		super().__init__(person_name)
+
+
+class Staff(Person):
+	""""A class for staff"""
+
+	def __init__(self, person_name):
+		super().__init__(person_name)
+	
```

Comment:
In both classes, you inherit from super class by `super(Staff, self).__init__(*args, **kwargs)` hence no need to rewrite `def __init__(self, person_name):` but as `def __init__(self, *args, **kwargs):`